src/main/kotlin/com/epam/brn/dto/StudyHistoryDto.kt

```diff
@@ -0,0 +1,25 @@
+package com.epam.brn.dto
+
+import java.time.LocalDateTime
+import javax.validation.constraints.NotNull
+
+class StudyHistoryDto(
+    @NotNull
+    val userId: Long,
+    @NotNull
+    val exerciseId: Long,
+    @NotNull
+    val startTime: LocalDateTime,
+    @NotNull
+    val endTime: LocalDateTime,
+    @NotNull
+    val doneTasksCount: Short,
```

Comment:
What do you think of use here something like default values like 0 for doneTasksCount, successTasksCount to let frontend skip this in dto by creation?

Reply:
Implementing...

Reply:
Done